internal/runbits/requirements/requirements.go

```diff
@@ -297,6 +297,12 @@ func (r *RequirementOperation) ExecuteRequirementOperation(
 	//	return errs.Wrap(err, "Could not get remote build expr")
 	//}
 
+	// refresh or install runtime
+	err = runbits.RefreshRuntime(r.Auth, r.Output, r.Analytics, r.Project, commitID, true, trigger, r.SvcModel)
+	if err != nil {
+		return handleRefreshError(err, r.Project, parentCommitID)
```

Comment:
If this is not a build error, the commit ID will be updated, but the buildscript will not be updated on line 310 below (commented for now, but it will eventually be uncommented when local orders are restored).

Perhaps we need to break out lines 306-315 into a separate function and call that function on line 306 and inside `handleRefreshError()`.